Frameworks/UIKit/UISegment.mm

```diff
@@ -36,53 +36,56 @@ + (instancetype)initialize {
 }
 
 - (instancetype)initWithCoder:(NSCoder*)coder {
-    id result = [super initWithCoder:coder];
-    id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
-    _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
+    if (self = [super initWithCoder:coder]) {
+        id info = [[coder decodeObjectForKey:@"UISegmentInfo"] retain];
+        _position = [coder decodeInt32ForKey:@"UISegmentPosition"];
 
-    CGRect frame;
-    frame = [self bounds];
+        CGRect frame;
+        frame = [self bounds];
 
-    if ([info isKindOfClass:[NSString class]]) {
-        _title = info;
-    } else if ([info isKindOfClass:[UIImage class]]) {
-        _image = info;
-    } else {
-        assert(0);
+        if ([info isKindOfClass:[NSString class]]) {
+            _title = info;
+        } else if ([info isKindOfClass:[UIImage class]]) {
+            _image = info;
+        } else {
+            assert(0);
```

Comment:
Nit - no need to fix now but we're trying to move away from assert(0) and replace with FAIL_FAST_MSG etc.